src/main/flight/autopilot.h

```diff
@@ -19,12 +19,20 @@
 
 #include "pg/autopilot.h"
 #include "flight/pid.h"
+#include "io/gps.h"
```

Comment:
I've removed the call to the config so I can just use references to the pg where access to the config is needed, rather than including the whole file.

Not sure if that's the right way but it keeps things separate.

Reply:
Lots of changes, able to only include pg/ files rather than whole headers - see new commit.